18_UIImageView — UIScrollView, WKWebView/16_UIImageView/WebViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  WebViewController.swift
+//  16_UIImageView
+//
+//  Created by admin on 13.07.2021.
+//
+
+import UIKit
+import WebKit
+
+class WebViewController: UIViewController, WKUIDelegate {
```

Comment:
```suggestion
class WebViewController: UIViewController {
```
WKUIDelegate  - всегда в extension